source/Plugins/ExpressionParser/Swift/SwiftASTManipulator.cpp

```diff
@@ -701,12 +701,6 @@ void SwiftASTManipulator::MakeDeclarationsPublic() {
                 llvm::dyn_cast<swift::AbstractStorageDecl>(decl))
           var_decl->overwriteSetterAccess(access);
       }
-
-      // FIXME: Remove this once LLDB has proper support for resilience.
-      if (swift::VarDecl *var_decl =
```

Comment:
Cleanup suggestion: can you either file a radar on me, or prepare a patch to remove VarDecl::setREPLVar()? I don't believe its used for anything else.